src/transformers/optimization.py

```diff
@@ -604,119 +604,14 @@ def scheduler_hook(param):
     )
 
 
-class AdamW(Optimizer):
+class AdamW(TorchAdamW):
```

Comment:
as long as you are sure all paramrs are the same for init, let'sgo! 🤗

Reply:
we can also deprecate this one?

Reply:
Good point, actually - the old one was already deprecated! Maybe we should just remove it entirely, since we've been showing a deprecation warning for a long time now?